src/lib/document-handler/builder.ts

```diff
@@ -0,0 +1,21 @@
+import type { Config } from '../../types/config'
+import buildGenerator from '../key-generators/builder'
+import buildStore from '../document-stores/builder'
+import DocumentHandler from "./index"
```

Comment:
```suggestion
import DocumentHandler from './index'
```

I think it'd be nice to lint/format the files